src/content/certifications/programs/infrastructure-automation.json

```diff
@@ -32,6 +32,6 @@
 				"register": "https://cp.certmetrics.com/hashicorp/en/login"
 			},
 			"faqSlug": "terraform-associate-002"
-		}	
+		}
```

Comment:
🗒️ I think this change happened during the commit hook when I merged `main` into this branch.